emails/tests/models_tests.py

```diff
@@ -175,18 +179,12 @@ def test_delete_mozmail_deleted_address_object(self):
         ).count()
         assert deleted_count == 1
 
-    # trigger a collision by making address_default always return 'aaaaaaaaa'
-    @override_settings(RELAY_FIREFOX_DOMAIN='default.com')
-    @patch.multiple('string', ascii_lowercase='a', digits='')
-    @patch('emails.models.DOMAINS', TEST_DOMAINS)
-    def test_make_relay_address_doesnt_make_dupe_of_deleted(self):
-        test_hash = sha256('aaaaaaaaa'.encode('utf-8')).hexdigest()
-        DeletedAddress.objects.create(address_hash=test_hash)
-        try:
-            RelayAddress.make_relay_address(self.user_profile, domain='default.com')
-        except CannotMakeAddressException:
-            return
-        self.fail("Should have raise CannotMakeAddressException")
+    def test_valid_address_dupe_of_deleted_invalid(self):
+        relay_address = RelayAddress.objects.create(user=baker.make(User))
+        relay_address.delete()
+        assert not valid_address(
+            relay_address.address, get_domain_from_number(relay_address.domain)
```

Comment:
```suggestion
            relay_address.address, relay_address.domain_value
```
should get you the domain's string equivalent value.

Reply:
`domain_value` returns the string of what the numerical value would be. I used it to generate the [full_address](https://github.com/mozilla/fx-private-relay/blob/main/emails/models.py#L340-L341) which is used in templates.